src/components/shared/LanguageSelector/LanguageSelector.test.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { render, cleanup, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import LanguageSelector from './LanguageSelector';
+import DopplerIntlProvider from '../../../i18n/DopplerIntlProvider.double-with-ids-as-values';
+import { AppServicesProvider } from '../../../services/pure-di';
+import { Router } from 'react-router-dom';
+import { MemoryRouter } from 'react-router-dom';
+
+const defaultDependencies = {
+  window: {
+    MenuButton: {},
+  },
+};
+
+describe('Language Selector Component', () => {
```

Comment:
Is it possible testing this scenario?

1. Open a page with redirect parameter, for example: `/signup?redirect=test` (and `/signup?another&redirect=test`, and other weird cases)
2. Click in `ES` (or `EN`)
3. Verify if the new URL has the expected redirect parameter.

Reply:
i add more cases now

Reply:
Si, pero no veo ningún caso donde haya navegación, es decir ese test que te pasé no está, no? o se me escapa algo?

Reply:
Perdon se me paso esto. Voy a investigar